### PR TITLE
New feature in the fextract tool to directly export in the csv format

### PR DESCRIPTION
**Pull request by emotheau**
## Summary
This PR adds a feature to the fextract tool to directly export in the csv format

## Checklist

The proposed changes:
- [ ] fix a bug or incorrect behavior in AMReX
- [X] add new capabilities to AMReX
- [ ] changes answers in the test suite to more than roundoff level
- [ ] are likely to significantly affect the results of downstream AMReX users
- [ ] are described in the proposed changes to the AMReX documentation, if appropriate

### CODE REVIEW

**Comment by asalmgren** [comment]
Emmanuel -- this would be a great opportunity to document some of the tools such as fextract and fcompare.  Could you add to this PR a short section of amrex documentation to describe what fextract now does?

File: Tools/Plotfile/fextract.cpp [hunk 1]
```diff
@@ -42,6 +43,8 @@ void main_main()
             fine_level = std::stoi(amrex::get_command_argument(++farg));
         } else if (name == "-e" || name == "--scientific") {
             scientific = true;
+        } else if (name == "-csv" or name == "--csv") {
```
Comment (WeiqunZhang) [inline Tools/Plotfile/fextract.cpp]:
```suggestion
        } else if (name == "-csv" || name == "--csv") {
```

**Comment by emotheau** [comment]
@asalmgren Sure I will do it. Let me some time to finish other things before !

**Comment by emotheau** [comment]
This PR is complete, could someone merge it in development if everything is ok?

**Review by WeiqunZhang — APPROVED** [review]

**Pull request merged**